src/System.Runtime.Extensions/src/System.Runtime.Extensions.csproj

```diff
@@ -176,6 +176,35 @@
       <Link>Common\Interop\Windows\mincore\Interop.LookupAccountNameW.cs</Link>
     </Compile>
   </ItemGroup>
+  <ItemGroup Condition=" '$(TargetsWindows)' == 'true' And '$(CoreClrOrCorRt)' == 'true'"> <!-- Microsoft.Win32.Registry, needed for some Environment env var support -->
+    <Compile Include="..\..\Microsoft.Win32.Registry\src\Microsoft\Win32\Registry.cs" />
```

Comment:
This seems like a lot code. Any data on how much it adds to the size of the output?